packages/components/src/mobile/bottom-sheet/range-cell.native.js

```diff
@@ -20,14 +25,38 @@ export default function BottomSheetRangeCell( props ) {
 		step = 1,
 		minimumTrackTintColor = '#00669b',
 		maximumTrackTintColor = Platform.OS === 'ios' ? '#e9eff3' : '#909090',
-		thumbTintColor = Platform.OS === 'ios' ? '#fff' : '#00669b',
+		thumbTintColor = Platform.OS === 'android' && '#00669b',
 		...cellProps
 	} = props;
+	const [ accessible, onChangeAccessible ] = useState( true );
+
+	const onCellPress = () => {
+		onChangeAccessible( false );
+		if ( this.sliderRef ) {
+			const reactTag = findNodeHandle( this.sliderRef );
+			AccessibilityInfo.setAccessibilityFocus( reactTag );
+		}
+	};
+
+	const accessibilityLabel =
+	sprintf(
+		/* translators: accessibility text. Inform about current spacer height value. %2$s: Spacer height current value. */
```

Comment:
RangeCell is not limited to Spacer block so we are not supposed to put block specific description here.

Reply:
Make sense, thanks!